apps/api/migrations/README.md

```
@@ -38,19 +38,35 @@ migrate -path apps/api/migrations -database "$DATABASE_DSN" down N
 
 ## Adding a New Migration
 
+Claim your number when you **open** the PR, not when you start work. Several
+PRs developed in parallel each grabbed what was "the next" number at the time,
+which is how the collisions at 060/061/069/070/081 happened.
+
 1. Find the next available sequential number:
    ```bash
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)